voresource/date_role/terms.csv

```diff
@@ -10,3 +10,6 @@ Submitted;1;Submitted;"The date the creator submits the resource to the publishe
 Updated;1;Updated;"The date of the last update to the resource;  when the resource is being added to."
 update;1;Update;A date on which the resource was updated (deprecated in favour of Updated);ivoasem:useInstead(Updated) ivoasem:deprecated
 Valid;1;Valid;A date of validitiy.  This is probably not useful in the VOResource context.
+
+# VEP-012
+Validated;1;Last validated;Dates with this role indicate when the resource has last undergone a non-formal validation.  This will usually involve a human ensuring that the resource still works as expectable, both technically and as regards science content.  The prototypical case is when tutors ensure a written tutorial still describes the real behaviour of clients and services.;ivoasem:preliminary
```

Comment:
could this line be splitted for better readability?

Reply:
Well, it could, but then it wouldn't be CSV any more, at least not something that all CSV parsers can read.  For now, I'm rather convinced letting people CSV-process the terms files may be useful one of these days.

But given things have developed in a direction where only the vocabulary maitainer actually plays with these files, we might reconsider; I'll open up Vocabularies 1.1 soon-ish.

Reply:
Oh, right. Silly comment on my side.